### ISSUE TITLE
[BUG] DlChip - Max width behaves like width.

### ISSUE THREAD

**Issue by fadiDL** [dataloop-ai/components#300]
When passing max width 300px it will override the width of the text entirely and set the chip to 300...

**Comment by fadiDL** [comment]
This also happens in DlMenu component. needs attention also there

**Comment by vadim-popa-opentech** [comment]
![image](https://github.com/dataloop-ai/components/assets/127128115/a20d4a9b-1841-4e3c-8e1f-d9126e5b0f97)

![image](https://github.com/dataloop-ai/components/assets/127128115/babfd96d-a5df-4baf-a3d8-09e35d8e7ec0)

![image](https://github.com/dataloop-ai/components/assets/127128115/3936c374-626a-4f07-a067-4b7807ab2324)

![image](https://github.com/dataloop-ai/components/assets/127128115/af1fd97c-6052-407a-98b6-2b2b3bcf027e)

@fadiDL, it cannot be reproduced. Can you provide more information, maybe I missed something

**Comment by fadiDL** [comment]
Youre right @vadim-popa-opentech closing ticket as not reproduced anymore
